homeassistant/components/camera/__init__.py

```diff
@@ -69,6 +77,12 @@
     vol.Required(ATTR_FILENAME): cv.template
 })
 
+CAMERA_SERVICE_PLAY_STREAM = CAMERA_SERVICE_SCHEMA.extend({
+    vol.Required(ATTR_MEDIA_PLAYER): cv.entities_domain(DOMAIN_MP),
+    vol.Optional(ATTR_FORMAT, default='hls'): vol.In(OUTPUT_FORMATS),
+    vol.Optional(ATTR_KEEPALIVE, default=False): cv.boolean,
```

Comment:
It feels weird to have keepalive be an option in the service. It allows users to shoot themselves in the foot if they have a camera that generates a new endpoint each time (like us). I suggest that we move it.